clusterloader2/pkg/measurement/common/ooms_tracker.go

```diff
@@ -55,13 +56,14 @@ func createClusterOOMsTrackerMeasurement() measurement.Measurement {
 }
 
 type clusterOOMsTrackerMeasurement struct {
-	selector  *measurementutil.ObjectSelector
-	msgRegex  *regexp.Regexp
-	isRunning bool
-	startTime time.Time
-	stopCh    chan struct{}
-	oomsLock  sync.Mutex
-	ooms      []oomEvent
+	selector      *measurementutil.ObjectSelector
+	msgRegex      *regexp.Regexp
+	isRunning     bool
+	startTime     time.Time
+	stopCh        chan struct{}
+	oomsWhitelist []string
```

Comment:
just whitelist?
now oomsLock is ambiguous

Reply:
Done.